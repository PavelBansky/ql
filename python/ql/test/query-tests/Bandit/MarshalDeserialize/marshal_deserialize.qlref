Bandit/marshal_deserialize.ql