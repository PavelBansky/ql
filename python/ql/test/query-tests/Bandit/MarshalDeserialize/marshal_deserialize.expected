| Test.py:6:7:6:31 | Attribute() | Deserialization with the marshal module is possibly dangerous. |
| Test.py:11:7:11:28 | Attribute() | Deserialization with the marshal module is possibly dangerous. |
